docs/manual/.gitignore

```diff
@@ -1,6 +1,9 @@
 # generated files
+/debug.txt
```

Comment:
Can you add information about which files are generating these ignored files? So that we know in the future if we have to remove them.

Reply:
done